config/config_test.go

```diff
@@ -88,20 +79,14 @@ func testValidateTesthelper(
 		cfgService.Attributes["map_rate_sec"] = -1
 		_, err = newConfig(cfgService)
 
-		if !cloudStoryEnabled {
-			test.That(t, err, test.ShouldBeError, newError("cannot specify map_rate_sec less than zero"))
-		} else {
-			test.That(t, err, test.ShouldBeNil)
-		}
+		test.That(t, err, test.ShouldBeError, newError("cannot specify map_rate_sec less than zero"))
 	})
 
 	t.Run(fmt.Sprintf("All parameters e2e%s", suffix), func(t *testing.T) {
 		cfgService := makeCfgService(false, cloudStoryEnabled)
 		cfgService.Attributes["sensors"] = []string{"a", "b"}
 		cfgService.Attributes["data_rate_msec"] = 1001
-		if !cloudStoryEnabled {
-			cfgService.Attributes["map_rate_sec"] = 1002
-		}
```

Comment:
test revert to reflect config validation revert so that we can make config changes without breaking cartographer